src/storage_utils.cpp

```diff
@@ -63,19 +63,20 @@ namespace libtorrent { namespace aux {
 		return ret;
 	}
 
-	typed_span<iovec_t> advance_bufs(typed_span<iovec_t> bufs, int const bytes)
+	typed_span<iovec_t> advance_bufs(typed_span<iovec_t> const& bufs, int const bytes)
 	{
```

Comment:
ok, maybeI can get some help to look at it. I will post some more details of what I see in the main comments